conf/default.yaml

```diff
@@ -0,0 +1,20 @@
+debug: true
+s3:
+  Bucket: '{{ENV.BUCKET}}'
```

Comment:
These are reserved case-sensitive keynames used by aws-sdk. If we lowercase them in default.yaml we will still have to uppercase them when the method is called, before we can pass them to AWS.  I'm cool with it either way, let me know if you still want me to change this @jgallen23